AzureDataLake/Scripts/ProcessCCB.usql

```diff
@@ -1,47 +1,45 @@
-﻿USE DATABASE [DecisionServiceDBTest];
+﻿SET @@FeaturePreviews = "DataPartitionedOutput:on";
+
+USE DATABASE [DecisionServiceDBTest];
 
 REFERENCE ASSEMBLY DecisionServiceDBTest.[DecisionServiceExtractor];
 
-DECLARE @_appId string = "xbetdealshomespotlight";    //your appId
-DECLARE @_account string = "decisionengine";    // your account name
+DECLARE @_appId string = "";    //your appId
+DECLARE @_account string = "";    // your account name
 
-DECLARE @_year string = "2019";  //your date: year
-DECLARE @_month string = "10";     //your date: month
-DECLARE @_day string = "14";       //your date: day
+DECLARE @_start DateTime = new DateTime(2019, 10, 1);  //your inclusive start date
+DECLARE @_end DateTime = new DateTime(2019, 10, 12);    //your non-inclusive end date
 
-DECLARE @OutputFolder string = "/" + @_appId + "_daily/";      //your output folder
-DECLARE @OutputSlimLogs bool = false;     //true if you want to have slime version of logs (all metadata without features)
+DECLARE @OutputFolder string = "/daily/" + @_appId + "/";      //your output folder
+DECLARE @OutputSlimLogs bool = true;     //true if you want to have slime version of logs (all metadata without features)
```

Comment:
Is OutputSlimLogs used? I see that `IF @OutputSlimLogs == true THEN` below was removed

Reply:
no, removed for now.